.editorconfig

```diff
@@ -57,3 +60,36 @@ csharp_style_allow_blank_line_after_colon_in_constructor_initializer_experimenta
 # IDE0073: The file header is missing or not located at the top of the file
 file_header_template = Licensed to the .NET Foundation under one or more agreements.\nThe .NET Foundation licenses this file to you under the MIT license.
 dotnet_diagnostic.IDE0073.severity = error
+
+# CA1010: Generic interface should also be implemented
+dotnet_diagnostic.CA1010.severity = none
+
+# CA1305: Specify IFormatProvider
+dotnet_diagnostic.CA1305.severity = none
+
+# CA1710: Identifiers should have correct suffix
+dotnet_diagnostic.CA1710.severity = none
+
+# CA1720: Identifier contains type name
+dotnet_diagnostic.CA1720.severity = none
+
+# CA1725: Parameter names should match base declaration
```

Comment:
Why do we disable this?  Is it because we have instances where we don't do it and there are too many to suppress with pragma at the site?

Reply:
I only found one case so far, but my thinking was that if we are migrating more assemblies that have this issue, I can't just change the name of the parameter in a public API. So I thought it would be best to just suppress it globally. Do you think I should use the pragma instead?

Reply:
I guess if we set it in runtime it's ok here.  I wouldn't want this set for new code but this repo isn't getting any new code so 🤷‍♂️

Reply:
Note that [in runtime](https://github.com/dotnet/runtime/blob/16cb41496d595e2568574cfe11c763d5e05136c9/eng/CodeAnalysis.src.globalconfig#L314) we use `suggestion`. I set it to `none` on purpose here.